### DISCUSSION TITLE
Releases?

### DISCUSSION THREAD

**Discussion by catgirldoll** [calloatti/modrobot#20] [General]
As the wiki directs users to download this program via the releases and there are no releases listed, I was wondering if there are any plans to publish in the near future?

**Comment by jerbear2008** [comment]
@calloatti Could you look at this?

**Comment by calloatti** [comment]
https://github.com/calloatti/modrobot/releases/tag/v5.0.0

**Comment by calloatti** [comment]
The releases were taken down as a precaution when Overwolf purchased CurseForge. Now that a CF API key has been provided by OW, there is a new release. It took a while, sorry about that.
